www/src/components/ReactPlayground.js

```diff
@@ -25,8 +25,8 @@ import * as yup from 'yup';
 import useEventCallback from '@restart/hooks/useEventCallback';
 import useIsomorphicEffect from '@restart/hooks/useIsomorphicEffect';
 import useMutationObserver from '@restart/hooks/useMutationObserver';
-import PlaceholderImage from './PlaceholderImage';
```

Comment:
Didn't see these files changed.  Do these imports need to be changed?

Reply:
I was going to ask about this in a comment on the PR, but it slipped my mind. When I tried to create a commit, eslint was throwing the `no-named-as-default` rule violation on these imports for some reason.

Reply:
Might be a false positive.  Could you revert this change?  This import wouldn't work

Reply:
Yes. There are also a couple of the example files that use these imports. I will change those to use the default import rather than the named import, as well.

Reply:
@kyletsang I'd kind of like to iron out this commit error issue so I can more easily contribute in the future. Would you mind taking a look at this error output (see attached file), and see if you can tell what's going on? I have not changed any config or rc files in my forked version of the repo, so not sure why it's just my local environment causing this. Much appreciated.
[commit_error_output.txt](https://github.com/react-bootstrap/react-bootstrap/files/8865380/commit_error_output.txt)

Reply:
Just checked my end and looks like we have an issue with our config. We haven't updated the doc components in a while, so this probably broke sometime long ago.

You can skip the precommit step for now.  I'll take a look at fixing this issue when I get a chance.